src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,152 @@ directly to CMake; for example::
-| ENABLE_WARNINGS_AS_ERRORS    | OFF     | Compiler warnings treated as errors    |
-|                              |         | errors.                                |
+| ENABLE_CUDA                  | OFF     | Enable CUDA                            |
++------------------------------+---------+----------------------------------------+
+| ENABLE_HIP                   | OFF     | Enable HIP                             |
 +------------------------------+---------+----------------------------------------+
 
-If ``AXOM_ENABLE_ALL_COMPONENTS`` is OFF, you must explicitly enable the desired
-components (other than 'core', which is always enabled).
-
-See `Axom software documentation <../../../index.html>`_
-for a list of Axom's components and their dependencies.
+Note that, in most Axom components, node-level parallelism features, enabled with 
+OpenMP, CUDA (NVIDIA GPUs), and HIP (AMD GPUs), are implemented using RAJA. See
+:ref:`dependencies-label` for instructions to set the directory location of a
+RAJA installation. In addition, enabling such features may require additional build 
+options to be provided, which are summarized in the following table.
+
++------------------------------+------------------------------------------+
+| OPTION                       | Description                              |
++==============================+==========================================+
+| RAJA_DIR                     | RAJA installation directory as described |
+|                              | in :ref:`dependencies-label`.            |
+|                              |                                          |
+|                              | RAJA must be built with support enabled  |
+|                              | for OpenMP, CUDA, or HIP to use features |
+|                              | enabled by those parallel programming    |
+|                              | models in Axom.                          |
++------------------------------+------------------------------------------+
+| CUDA_TOOLKIT_ROOT_DIR        | Path to CUDA software stack installation |
++------------------------------+------------------------------------------+
+| CMAKE_CUDA_COMPILER          | Path to CUDA compiler (e.g. nvcc)        |
++------------------------------+------------------------------------------+
+| CMAKE_CUDA_ARCHITECTURES     | Target architecture(s) for CUDA          |
++------------------------------+------------------------------------------+
+| HIP_ROOT_DIR                 | Path to HIP software stack installation  |
++------------------------------+------------------------------------------+
+| CMAKE_HIP_ARCHITECTURES      | Target architecture(s) for HIP           |
++------------------------------+------------------------------------------+
+
+.. important:: Depending on the version of RAJA you are using, you may need to
+               set the C++ standard to C++14. RAJA requires C++14 as of release
+               v2022.03.0.
```

Comment:
Maybe this change is outside the PR's scope, but if Axom requires C++14 (as @white238 pointed out), shouldn't we ensure the C++ standard is at least C++14?

Reply:
We do that here: https://github.com/LLNL/axom/blob/develop/src/CMakeLists.txt#L46 but it doesn't cover all app integration cases.  But, we have this guard here for users:
https://github.com/LLNL/axom/blob/develop/src/CMakeLists.txt#L100

Reply:
Done.